### ISSUE TITLE
Click not working for Openlayers 3 element in Windows 10 LTSB: IE11

### ISSUE THREAD

**Issue by karniemi** [SeleniumHQ/selenium#6742]
## 🐛 Bug Report

In Windows 10 LTSB and IE11, clicking with selenium on Openlayers 3 canvas element does not produce the same result as human originated mouse click. With a selenium originated click, nothing happens( it's not random but consistently nothing happens). With Windows 7 & IE11 the same click was working OK -with the very same test code, and selenium&ie-driver versions. Also, with CEF, in both W7 and W10 the click is working.

We found a workaround for this problem:
* by switching IE to the legacy event model the click started working ( MS changed the event model in W8, more about this can be found on www)
* the registry setting is FEATURE_NINPUT_LEGACYMODE

## To Reproduce

TDB. I will ask for a minimal reproduction set from  our devs. Please close this ticket until we provide this information, if that's your policy. (At this point my main contribution plan was to first share the finding and the workaround that we found.)

DesiredCapabilities:
* NATIVE_EVENTS: default
* ENABLE_PERSISTENT_HOVERING: false

Detailed steps to reproduce the behavior:
TBD.

## Expected behavior

Clicking with selenium should produce the same result as a real click.

## Test script or set of commands reproducing this issue

TBD

## Environment

OS: Windows 10 LTSB
Browser:  IE11
Browser version: 
Browser Driver version:  IEDriverServer: 3.2.0.0
Language Bindings version: selenium-java: 3.2.0

We also tested with latest IEDriverServer and selenium-java with no effect (, other than massively slower test execution.)

**Comment by jimevans** [comment]
Can you please try setting the `requireWindowFocus` capability to true (without the registry workaround)?

**Comment by karniemi** [comment]
Hi, my colleague tested setting `requireWindowFocus:true` and disabling the registry workaround: the clicks seemed to be working.

We would not like to take the `requireWindowFocus`-path... because of the very same downsides that have been already documented by the selenium project:  the devs are running tests on their dev machines and working with something else at the same time -and also,  we have tests that use two iedriver instances to test some interactions between two users of the system.

It's also a bit odd that in W7 we did not need `requireWindowFocus`.

Anything we can do to help finding/fixing the actual root cause of this problem?

**Comment by karniemi** [comment]
BTW, while troubleshooting this issue originally,  I did setup some event monitors in IE to see the mouse-events that are passed to the DOM-object. The mouse click and mouseup events originated from selenium clicks were somewhat different than real clicks. For example, "buttons" had value:1 - though according to specs it should probably have value:0. Reading the specs and discussion about "buttons", I would interpret it to indicate which buttons we still down when the event was fired...so the value:1 seems a bit odd because in click and mouseup that button was already released. Some true long-shot guessing, though.  I also tried manipulating the event in an eventlistener...but  buttons-field is read-only so I could not do it.
